docs(fix): vpc direct connect MTA-1975 (#618)
scaleway/docs-content: faq/enterprise.mdx

This new range of Instances is the most powerful range of Instances to ever be offered by Scaleway, from 8 vCPU and 32 GB of RAM (ENT1-S) to 96 vCPU & 384GB of RAM (ENT1-2XL) for the most demanding workloads.

> Enterprises Instances will be compatible with our ecosystem: from our Free and Fully-Managed Kubernetes Control Plane to our VPC features such as Private Networks and Public Gateway.

Reviewer: ```suggestion
Enterprise Instances are fully compatible with our ecosystem: from our free and fully-managed Kubernetes control plane to our VPC features such as Private Networks and Public Gateway.
```
